planner/core/explain.go

```diff
@@ -542,7 +542,7 @@ func (p *PhysicalWindow) ExplainInfo() string {
 		}
 		buffer.WriteString("order by ")
 		for i, item := range p.OrderBy {
-			order := "asc"
+			var order string
```

Comment:
There will be an extra space that you should remove it.

Reply:
updated